cloudinit/sources/DataSourceAzure.py

```diff
@@ -364,14 +365,33 @@ def _get_subplatform(self):
             subplatform_type = "seed-dir"
         return "%s (%s)" % (subplatform_type, self.seed)
 
+    @azure_ds_telemetry_reporter
+    def _check_if_primary(self, ephipv4: EphemeralIPv4Network) -> bool:
+        if not ephipv4.static_routes:
+            # Primary nics must contain routes.
+            return False
+
+        routed_networks = [r[0] for r in ephipv4.static_routes]
+        wireserver_route = f"{self._wireserver_endpoint}/32"
+        primary = any(
+            n in routed_networks
+            for n in [
+                "168.63.129.16/32",
```

Comment:
Isn't this the same as the default `wireserver_route`? Why duplicate?

Reply:
You are correct, it should be

Reply:
Nice, thanks!